src/sass/_gallery.scss

```diff
@@ -0,0 +1,13 @@
+.gallery {
+  background-image: linear-gradient(
+    to bottom,
+    $bottom-section-background 0%,
+    $bottom-section-background 45%,
+    $body-background 45%,
+    $body-background 100%
+  );
+  margin-bottom: 40px;
```

Comment:
Wydaje mi się, że tu powinien być margin 80px, żeby było tak, jak na makiecie. Albo niższej sekcji może dać margin-top 40px i wyjdzie na to samo :)

Reply:
Zrobione, rzeczywiście moje niedopatrzenie.